pkg/translator/jaeger/constants.go

```diff
@@ -25,6 +25,10 @@ const (
 	statusOk    = "OK"
 )
 
+// eventNameAttr is a Jaeger log field key used to represent OTel Span Event Name as defined by the OpenTelemetry Specification:
+// https://github.com/open-telemetry/opentelemetry-specification/blob/34b907207f3dfe1635a35c4cdac6b6ab3a495e18/specification/trace/sdk_exporters/jaeger.md#events
+const eventNameAttr = "event"
```

Comment:
Should we make this a featuregate protected change?

Reply:
We could. I just don't think this change is that impactful. WDYT? Should we?

Reply:
ok... let's do it this way